### PR TITLE
Made some Minor Styling Changes Across App

### PR DESCRIPTION
**Pull request by cnnebe**
I made some changes to the app based on what we discussed last week, and to try to make things more consistent. I changed the mountain image, changed the containers across the app, changed the login image, changed the the terms of service and privacy policy, expanded the index page, modified the home page cards, and made a few other minor changes.

### CODE REVIEW

File: HandprintGenerator/static/css/handprintgenerator.css [hunk 1]
```diff
@@ -80,7 +88,7 @@ h2 {
 
 /* Carousel base class */
 .carousel {
-  max-height: 500px;
+  max-height: 700px;
```
Comment (jinjennifer) [inline HandprintGenerator/static/css/handprintgenerator.css]: Do you think you can make the image smaller in height? On my screen, with the increase in height, you can only see the image, and nothing else, not even the white text overlaying the image. I feel like it is important to be able to get some information about Handprinter at first glance.

File: HandprintGenerator/templates/HandprintGenerator/detail.html [hunk 2]
```diff
@@ -53,8 +52,8 @@ <h2 data-toggle="tooltip" title="The title of the action idea."><b>{{ ai.name }}
                 <div class="leftalign">
                 {% if ai.creator == request.user or user.is_staff or user.profile.role == "Moderator" %}
                     {% if ai.active %}
-                    <a href="{% url 'HandprintGenerator:edit_action_idea' ai.id %}" class="btn btn-link ">Edit Action Idea</a>
-                    <a href="{% url 'HandprintGenerator:delete_action_idea' ai.id%}" class="btn btn-link">Delete Action Idea</a> 
+                    <a href="{% url 'HandprintGenerator:edit_action_idea' ai.id %}" class="btn btn-success btn-link" style="color: white;">Edit Action Idea</a>
+                    <a href="{% url 'HandprintGenerator:delete_action_idea' ai.id%}" class="btn btn-warning btn-link" style="color: white;">Delete Action Idea</a>
```
Comment (jinjennifer) [inline HandprintGenerator/templates/HandprintGenerator/detail.html]: Could you also make report idea a button?

File: HandprintGenerator/templates/HandprintGenerator/user_profile.html [hunk 3]
```diff
@@ -7,13 +7,13 @@
...
 <div class="user_profile">
-    <div class="container">
+    <div class="container" id="second">
         <div class="row ">
             <div class="col-xs-12 col-md-12 user-details">
                
 
                 <!-- Profile Image -->
-                <div class="user-image"><img id="profile-img" class="profile-img-card" src={% static 'img/Pear_Leaf.jpg' %} /></div> 
+                <div class="user-image"><img id="profile-img" class="profile-img-card" src={% static 'img/Pear_Leaf.jpg'%} alt="profile"/></div>
```
Comment (jinjennifer) [inline HandprintGenerator/templates/HandprintGenerator/user_profile.html]: The file was deleted, so this image doesn't work.

File: HandprintGenerator/static/css/handprintgenerator.css (hunk 1 above)
Comment (jramjohn) [inline HandprintGenerator/static/css/handprintgenerator.css]: Yeah, they should be able to see the two buttons immediately

**Comment by jinjennifer** [comment]
I'm going to close this pull request. I made some additional modifications on top of your modifications since I didn't want to merge it unless the report button was changed to a button just to make it consistent, so I'm going to push my changes.

**Pull request closed without merging**